openpype/hosts/blender/api/utils.py

```diff
@@ -433,17 +437,52 @@ def make_paths_absolute(source_filepath: Path = None):
     relative_datablocks = get_datablocks_with_filepath(absolute=False)
     remapped_datablocks = set()
     if source_filepath:
+        # Get last workfile representation
+        workfile_repre = max(
+            filter(
+                lambda r: r["context"].get("version") is not None,
+                list(
+                    get_representations(
+                        get_current_project_name(),
+                        context_filters={
+                            "asset": get_current_asset_name(),
+                            "family": "workfile",
+                            "task": {"name": get_current_task_name()},
+                        },
+                    )
+                )
+            ),
+            key=lambda r: r["context"]["version"],
+        )
```

Comment:
Isn't it some logic shared with the download last workfile? Can't we make it a common function?